frontend/styles/custom.css

```diff
@@ -1108,3 +1083,18 @@ md-input-container:not(.md-input-invalid).md-input-has-value .inputEmail {
     font-size: 11px;
     margin-left: 10px;
 }
+
+.flat-btn {
```

Comment:
these should really be here?

Reply:
I put it here because it can be used later as an alternative to de Angular Material button.